src/model/AirLoopHVACUnitarySystem_Impl.hpp

```diff
@@ -84,12 +84,14 @@ namespace model {
 
       boost::optional<Schedule> supplyAirFanOperatingModeSchedule() const;
 
+      bool hasHeatingCoil() const;  // For speed
```

Comment:
These are in the impl so I probably shouldn't care. (not that I do) Curious though, what is the purpose of this over just asking for the ::heatingCoil which is optional and therefore behaves a lot like a bool?